Library/Formula/mutt.rb

```diff
@@ -74,11 +74,11 @@ class Mutt < Formula
   end if build.with? "confirm-attachment-patch"
 
   def install
+    
     args = ["--disable-dependency-tracking",
             "--disable-warnings",
             "--prefix=#{prefix}",
             "--with-ssl=#{Formula['openssl'].opt_prefix}",
-            "--with-sasl",
```

Comment:
Does `gsasl` work on Linux? You could add `depends_on "gsasl" => :recommended unless OS.mac?`.